### DISCUSSION TITLE
Make highlighting for Elasticsearch optional

### DISCUSSION THREAD

**Discussion by ilyad91** [grafana/grafana#37036] [Data Sources]
Since version 7.5.0 of Grafana a word highlighting feature was added to the Elasticsearch plugin
https://github.com/grafana/grafana/pull/30293

**What would you like to be added**:
A toggle so that "highlight" feature of the Elasticsearch plugin can be disabled.

**Why is this needed**:
Highlighting search results on Elasticsearch queries has a potential performance/load cost on Elasticsearch.
This feature should be made optional.

Thank you

**Comment by dave-martinez** [comment]
Adding to this discussion as well. 

Is there a way to disable highlighting for Elasticsearch datasource? 

Reasoning for me is if I only search for logs like `app:grafana` in a "Grafana infrastructure" dashboard, I have no need of highlighting since it's already implied that the logs is regarding Grafana.

Thank you!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
